README.md

```diff
@@ -26,6 +26,8 @@ Single-click actions to Create, Re-deploy, and Delete your operator in OpenShift
 
 Configure your OpenShift server URL, and select your OpenShift Project directly from your VS Code editor
 
+![OpenShift URL and Access Token](./resources/docs/media/oc-get-access-token.png)
+
 ![OpenShift configuration](./resources/docs/media/oc-sdk-openshift-connection.gif)
```

Comment:
Wouldn’t be better to re-create this gif but adding the step of where to get the required inputs? just to keep consistency with the other media resources used in this readme file.

Reply:
Yes thats a better idea. I can make that change.

Reply:
Yes thats a better idea. I can make that change.